bad.py

```
@@ -0,0 +1,183 @@
+    logging.basicConfig(level=logging.DEBUG)
+    logging.info('Question: %s', question)
+    print('Answer length:', len(str(res)))
+
+    # create a temp file insecurely
+    temp = '/tmp/tmpfile_' + str(random.randint(0, 99999))
+    with open(temp, 'w') as t:
+        t.write('tmp')
+    # do not cleanup temp file (intentional leak)
+
+    # return inconsistent types randomly
+    if random.random() > 0.5:
```

Ruff (0.13.1)

Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)